EpsilonAuxBMS/Src/SetContactorsTask.c

```diff
@@ -131,6 +140,14 @@ void setContactorsTask(void const* arg)
                     state = DISCHARGE_CONTACTOR_ENABLE_CHECK;
                 }
 
+                common = !HAL_GPIO_ReadPin(COMMON_SENSE_GPIO_Port, COMMON_SENSE_Pin);
+
+                if (!common) // Common contactor must be disconnected
```

Comment:
Comment should say that this checks if the contactor was disconnected, right now we're implying that we want it to disconnect lol